cmd/kube-proxy/app/server.go

```diff
@@ -89,6 +90,7 @@ import (
 
 func init() {
 	utilruntime.Must(metricsfeatures.AddFeatureGates(utilfeature.DefaultMutableFeatureGate))
+	logsapi.AddFeatureGates(utilfeature.DefaultMutableFeatureGate)
```

Comment:
Should there be a comment here suggesting that this should go away after `ContextualLogging` goes GA? Or will `logsapi.AddFeatureGates` just get removed entirely at that point which will then force it to be cleaned up everywhere?

Reply:
`logsapi.AddFeatureGates` will stay, no matter what happens to feature gates inside the package. It's basically part of the package's API.

/hold cancel

Reply:
But will it still do anything useful? Will we still want kube-proxy to keep calling it?